src/cdk-cli-integ-tests.ts

```diff
@@ -49,6 +68,20 @@ export interface CdkCliIntegTestsWorkflowProps {
    * @default - No upstream versions
    */
   readonly allowUpstreamVersions?: string[];
+
+  /**
+   * Invoke atmosphere service to retrieve AWS test environments.
+   *
+   * @default false
+   */
+  readonly atmosphereEnabled?: boolean;
+
+  /**
+   * Options for invoking the atmosphere service. Applicable only if `atmosphereEnabled` is true.
+   *
+   * @default { endpointVariable: 'ATMOSPHERE_ENDPOINT', oidcRoleArnVariable: 'ATMOSPHERE_OIDC_ROLE_ARN' }
+   */
+  readonly atmosphereOptions?: AtmosphereOptions;
```

Comment:
I don't like 2 properties for a single thing. Can we not say: `enableAtmosphere?: AtmosphereOptions` ?

Reply:
Done